tools/cli/commands/create.py

```diff
@@ -579,11 +592,13 @@ def run(args, gcloud_compute, gcloud_repos,
         cmd.extend(['--zone', args.zone])
 
     enable_backups = "false" if args.no_backups else "true"
+    idle_timeout = args.idle_timeout
     console_log_level = args.log_level or "warn"
     user_email = args.for_user or email
     service_account = args.service_account or "default"
     # We have to escape the user's email before using it in the YAML template.
     escaped_email = user_email.replace("'", "''")
+    initial_user_settings = '{{"idleTimeoutInterval": "{0}"}}'.format(idle_timeout) if idle_timeout else ''
```

Comment:
If we're building the initial settings here, the code should point out that this is JSON. I think we should be using `json.dumps` to make it obvious for future additions.